### PR TITLE
Add bDigg as collateral

### PR DESCRIPTION
**Pull request by smb2796**

### CODE REVIEW

File: UMIPs/umip-bdigg-collateral.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+| UMIP Title | Add bDigg as a collateral collateral currency |
+| Authors    | Sean Brown (@smb2796)  |
+| Status     | Draft                               |
+| Created    | April 7, 2021                    |
+
+## Summary
+
+This UMIP will add bDigg, the BadgerDAO DIGG vault LP token, as an approved collateral currency. 
+
+This will involve adding it to the whitelist and adding a flat final fee to charge per-request. The proposed final fee is X bDigg per request.
```
Comment (pemulis) [inline UMIPs/umip-bdigg-collateral.md:14]: Specified further down: `0.016`.

File: UMIPs/umip-bdigg-collateral.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+## Implementation
+
+This change has no implementation other than the aforementioned governor transactions.
+
+## Security considerations
+
+Digg is an elastic supply rebasing token that is intended to track the price of Bitcoin by adjusting the supply of Digg. Because of the rebasing mechanism, the price of Digg (and bDigg) can be prone to large amounts of volatility. The bDigg supply is not elastic in the same way as DIGG.
+
+To read more about this, view the Digg explainer [here](https://badger.finance/digg). Contract deployers and users should take care to adjust contract parameteriziation and usage to account for the potential volatility.
```
Comment (mhairimca) [inline UMIPs/umip-bdigg-collateral.md:52]: Looks good.

File: UMIPs/umip-bdigg-collateral.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
+# Headers
+
+| UMIP - 35 |                                     |
+| ---------- | ----------------------------------- |
+| UMIP Title | Add bDigg as a collateral collateral currency |
+| Authors    | Sean Brown (@smb2796)  |
+| Status     | Draft                               |
+| Created    | April 7, 2021                    |
+
```
Comment (mhairimca) [inline UMIPs/umip-bdigg-collateral.md:9]: Discourse Link?

Comment (smb2796) [reply UMIPs/umip-bdigg-collateral.md:9]: will add!

**Review by mhairimca — APPROVED** [review]

**Pull request merged**